src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -221,23 +221,35 @@ ScrollDown.propTypes = {
 
 export default function Banner() {
   const classes = useStyles();
-  const { version, telescopeUrl } = useSiteMetadata();
+  const { telescopeUrl } = useSiteMetadata();
   const [sha, setSha] = useState('');
+  const [version, setVersion] = useState('');
+  const [githubUrl, setGitHubUrl] = useState('');
 
-  async function getSha() {
+  async function getGitData() {
     try {
       const res = await fetch(`${telescopeUrl}/health`);
-      const data = await res.json();
-      return data.info.sha;
+
+      // Fetch failure
+      if (!res.ok) {
+        throw new Error(res.statusText);
+      }
+
+      // Checking whether content type is correct
+      if (res.headers.get('content-type').includes('application/json')) {
```

Comment:
Don't bother with this.  Ray did it because he was re-using the function, but we know it will be JSON.

Reply:
Removed.